### ISSUE TITLE
Training loss does not decrease

### ISSUE THREAD

**Issue by phuocnguyen2008** [junyanz/pytorch-CycleGAN-and-pix2pix#1595]
Dear authors @junyanz and @taesungp,

First, thank you for your excellent work!

I am currently using the pix2pix model to predict future images (gray-scale) based on the current one in the medical field.
I modified the code to use 7 input channels and 1 output channel for Generator.

During the training, I observed that the loss values of G_GAN, G_L1, D_Real, and D_Fake have not changed much since the beginning of the training process. (Figure 1)
![image](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/assets/34409468/192a9bfc-bd1d-4c6e-b7dd-265f1f351113)

Regarding the training results, the fake_B images are very different from the real_B ones.
![image](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/assets/34409468/7e10853f-dc4e-46a4-b5ad-7d09f339f504)

I am new to this task and any insights/explanations/recommendations are highly appreciated!
Thank you very much.

**Comment by Setarekhosravi** [comment]
Hello @phuocnguyen2008 
I have the same problem, did you get the answer?

**Comment by tongfh1** [comment]
Hello, how do you modify the code to realize the 7 input channels and 1 output channel?

**Comment by Parth527** [comment]
@tongfh1 there are flags `input_nc` and `output_nc` for that purpose
